pkg/controller/directvolumemigration/pvcs.go

```diff
@@ -119,6 +120,12 @@ func (t *Task) createDestinationPVCs() error {
 		err = destClient.Create(context.TODO(), &destPVC)
 		if k8serror.IsAlreadyExists(err) {
 			t.Log.Info("PVC already exists on destination", "name", pvc.Name)
+		} else if k8serror.IsForbidden(err) {
+			if strings.Contains(err.Error(), pvc.Name) && strings.Contains(err.Error(), "exceeded quota") {
```

Comment:
While this solves the problem, I don't like the idea of reading Error strings. We should just `Get()` the PVC and only attempt to create it when it's not present. Although it does add a call, it does not necessarily have to go to the api server.